pkg/client/client.go

```diff
@@ -510,3 +545,21 @@ func (client *DataSetClient) onBufferDrop(buf *buffer.Buffer, status uint32, err
 		)...,
 	)
 }
+
+func (client *DataSetClient) constructUserAgent() {
+	client.userAgent = fmt.Sprintf(
+		"id: %s, "+
```

Comment:
why dont we use unify syntax with common pattern `User-Agent: <product>/<version> (<system-information>) <platform> (<platform-details>) 
<extensions>` ? At least use `/` instead of `: `
eg. `datasetexporter/1.2.3 (...)`

note version of collector defines version of dataset-go library = no need to log it